src/axom/core/tests/core_array.hpp

```diff
@@ -1931,4 +1931,18 @@ TEST(core_array, checkVariadicCtors)
   Array<int, 3> arr12(s, s, s);
 }
 
+//------------------------------------------------------------------------------
+
+TEST(core_array, check_subspan_range)
+{
+  int m = 10;
+  int n = 3;
+  Array<int> arr(m);
+  ArrayView<int> arrv1(arr);
+  ArrayView<int> arrv2 = arrv1.subspan(n);
+  EXPECT_EQ(arrv2.size() + n, arrv1.size());
```

Comment:
Suggestion:
For clarity, it might be good to add some tests against the known sizes.
E.g.
```cpp
EXPECT_EQ(arr1.size(), arr.size());
EXPECT_EQ(arr2.size(), m-n);
```

Reply:
Agreed.